serverless.yml

```diff
@@ -8,6 +8,9 @@ package:
 environment:
   MONGO_USER: ${env:MONGO_USER}
   MONGO_PASSWORD: ${env:MONGO_PASSWORD}
+  AWS_ACCESS_KEY_ID: ${evn:AWS_ACCESS_KEY_ID}
+  AWS_SECRET: ${evn:AWS_SECRET}
+  AWS_USER_POOL_ID: ${evn:AWS_USER_POOL_ID}
```

Comment:
You have some typos in `env`

Reply:
typo fixed, the env variable was renamed to AWS_ACC_KEY_ID to avoid clashing with AWS/Serverless framework reserved config keyword